test/gurka/test_truck_restrictions.cc

```diff
@@ -49,6 +49,7 @@ TEST_P(TruckRestrictionTest, NotAllowed) {
   try {
     gurka::do_action(Options::route, map, {"A", "D"}, "truck",
                      {{"/costing_options/truck/" + option, v}});
+    FAIL() << "Expected no path to be found";
```

Comment:
this is just a small fix, hazmat was actually not set properly, so there was no exception thrown